### PR TITLE
[docs] Update README.md

### PR DESCRIPTION
**Pull request by Jennifer88huang-zz**
## Proposed changes

1. Categorize license related content in one section, and make it easier for users find the complete info.
2. Refine some sentences, making it more easier to read.
3. If someone finds a bug, the person has the potential to report issues or propose changes directly, add related info. 
## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Jennifer88huang** [comment]
Question: Do you have mailing list for users, such as user@@doris.apache.org?
If yes, you can add it in the **Contact us** section.

File: README.md [hunk 1]
```diff
@@ -36,40 +36,40 @@ Its original name was Palo, developed in Baidu. After donated to Apache Software
...
+> Some licenses of the third-party dependencies are not compatible with Apache 2.0 License. So you need to disable
+some Doris features to be complied with Apache 2.0 License. For details, refer to the `thirdparty/LICENSE.txt`
+
 ## 2. Technology
 Doris mainly integrates the technology of Google Mesa and Apache Impala, and it is based on a column-oriented storage engine and can communicate by MySQL client.
 
-## 3. User cases
-Doris not only provides high concurrent low latency point query performance, but also provides high throughput queries of ad-hoc analysis.
+## 3. Use cases
+- Doris provides high concurrent low latency point query performance, as well as high throughput queries of ad-hoc analysis.
```
Comment (zbtzbtzbt) [inline README.md:47]: I think it’s better to start this paragraph with this sentence “Features of Doris:”
or we can use a better subtitle to replace "Use cases"

Comment (Jennifer88huang) [reply README.md:47]: If you don't mind, I would like to include this part as "Highlights"(heading2) and put it in the "Apache Doris" section.

When the use case of Doris is ready, we can add this section later.

Comment (Jennifer88huang) [reply README.md]: I've applied the changes, and you can have a look at it.

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**